pkg/relayer/proxy/jsonrpc.go

```diff
@@ -157,7 +157,7 @@ func (jsrv *jsonRPCServer) serveHTTP(ctx context.Context, request *http.Request)
 	relayHTTPRequest := &http.Request{
 		Method: request.Method,
 		Header: request.Header,
-		URL:    &jsrv.proxiedServiceEndpoint,
+		URL:    jsrv.proxiedServiceEndpoint,
```

Comment:
Make sure to merge with main now that #189 has merged this is changed in name
```suggestion
		URL:    sync.proxiedServiceEndpoint,
```